pandas/tests/frame/test_axis_select_reindex.py

```diff
@@ -1151,3 +1151,24 @@ def test_raise_on_drop_duplicate_index(self, actual):
         expected_no_err = actual.T.drop('c', axis=1, level=level,
                                         errors='ignore')
         assert_frame_equal(expected_no_err.T, actual)
+
+    @pytest.mark.parametrize('index, drop_labels', [
+        ([1, 2, 3], []),
+        ([1, 1, 2], []),
+        ([1, 2, 3], [2]),
+        ([1, 1, 3], [1]),
+    ])
```

Comment:
I still think (see [my previous comment](https://github.com/pandas-dev/pandas/pull/21515#discussion_r195999517)) I would rather write this as:

```python
@pytest.mark.parametrize('index', [[1, 2, 3], [1, 1, 2]])
@pytest.mark.parametrize('drop_labels', [[], [1]])
```

Do you have any reason to explictly parametrize the combinations?

Reply:
Agree with you, your approach seems more compact. Let me update